rdr_service/tools/tool_libs/consent_error_report.py

```diff
@@ -80,17 +80,19 @@ def execute(self):
             # This enables use of SendGrid email service when running this tool from a dev server vs. app instance
             config.override_setting(config.SENDGRID_KEY, project_config[config.SENDGRID_KEY])
 
-        errors_since = self.args.errors_since
         report = ConsentErrorReportGenerator()
+        id_list = self.id_list
         # Specific ids will override any date filter
-        if self.id_list:
-            errors_since = None
-        report.create_error_reports(errors_created_since=errors_since,
+        if not id_list:
+            report.get_unreported_error_ids()
```

Comment:
Why is `get_unreported_error_ids` called here?

Reply:
Because a specific id list wasn't provided so it will default to generating the report for outstanding unreported errors.

Reply:
I may still be missing something, but it seems like `get_unreported_error_ids` doesn't store the list of ids on it's own (just returns them). Should the returned list be stored in something like `id_list`?